parm/archive/gfs_arcdir.yaml.j2

```diff
@@ -47,7 +47,7 @@
 
     {% if DO_JEDISNOWDA == True %}
         {% do det_anl_files.append([COMIN_SNOW_ANALYSIS ~ "/" ~ head ~ "snowstat.tgz",
-                                    ARCDIR ~ "/snowstat." ~ RUN ~ "." ~ cycle_YMDH ~ ".tgz"]) %}
+                                    ARCDIR ~ "/snowstat.tgz." ~ RUN ~ "." ~ cycle_YMDH ]) %}
```

Comment:
Shouldn't the extension be at the end of the filename?

Reply:
lol yes sorry